Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:16-alpine as buildstep
 ENV CI=true
 WORKDIR /workspace
 COPY package.json package-lock.json /workspace/
-RUN npm i --ignore-scripts
+RUN npm ci --prefer-offline --no-audit --ignore-scripts
```

Comment:
How can any `npm ci` be faster then `npm i`? Only if there is nothing to clean anyways, right?
Is the switch to `ci` in the Docker files intentionally?

Reply:
Yes, docker builds should always use `npm ci`, because then it will fail if the lock-file is not in sync with the package.json.
The `--no-audit` switch makes it a bit faster. And you are right: docker builds never have anything to clean.